server/rbacpolicy/rbacpolicy.go

```diff
@@ -25,6 +25,8 @@ const (
 	ActionDelete   = "delete"
 	ActionSync     = "sync"
 	ActionOverride = "override"
+
+	ActionActionType = "action"
```

Comment:
Unfortunately, this should be named ActionAction for consistency, and if we ever want to use reflection to get these values.